node/src/reactor/joiner.rs

```diff
@@ -871,14 +883,26 @@ impl reactor::Reactor for Reactor {
 
             Event::LinearChainAnnouncement(LinearChainAnnouncement::BlockAdded(block)) => {
                 let block_hash = *block.hash();
-                reactor::wrap_effects(
+                let mut effects = reactor::wrap_effects(
                     Event::EventStreamServer,
                     self.event_stream_server.handle_event(
                         effect_builder,
                         rng,
-                        event_stream_server::Event::BlockAdded { block_hash, block },
+                        event_stream_server::Event::BlockAdded {
+                            block_hash,
```

Comment:
Did I forget to remove that? It's redundant, because `block.hash()` doesn't re-compute the hash.

Reply:
Could be an issue caused by the merge. Let me fix that :+1:

Reply:
Ah, maybe I didn't actually change this event. There are multiple `BlockAdded`s.
Anyway, if it's easy, maybe we should remove the redundant block hash here, too.